business_requirement_deliverable_default/views/business_requirement_deliverable_default.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="product_template_only_form_view" model="ir.ui.view">
+        <field name="name">product.template.form</field>
+        <field name="model">product.template</field>
```

Comment:
As per the [file naming](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming) guideline of OCA, since the model name is `product.template`, the filename should be `product_template.xml`